### PR TITLE
[DOCS-399] API Documentation Style Guide

### PR DESCRIPTION
**Pull request by ana-dashuk-cobalt**
Hi team,

I'm submitting this style guide to provide basic writing guidelines and add more consistency to our API documentation. For context, please see API-650 and DOCS-319.

I'll submit another PR to add shortcodes and revise the current content.

### CODE REVIEW

File: APIDocumentationStyleGuide.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# API Documentation Style Guide
+
+At Cobalt, we follow the writing guidelines outlined in the [Google developer documentation style guide](https://developers.google.com/style), with the following exceptions:
+
+- We use title case in [document titles and headings](https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings). To check your title, use [this title capitalization tool](https://capitalizemytitle.com/).
```
Comment (zmomen-cobalt) [inline APIDocumentationStyleGuide.md:5]: nitpicky question 😄  ..  for the capitalization, which style to use? MLA, APA, chicago?

Comment (ana-dashuk-cobalt) [reply APIDocumentationStyleGuide.md]: @zmomen-cobalt Great question! We follow Chicago capitalization rules. My intent was to add this link `https://capitalizemytitle.com/style/Chicago/`, but this tool always lands me on the Chicago tab by default.

File: APIDocumentationStyleGuide.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# API Documentation Style Guide
+
+At Cobalt, we follow the writing guidelines outlined in the [Google developer documentation style guide](https://developers.google.com/style), with the following exceptions:
+
+- We use title case in [document titles and headings](https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings). To check your title, use [this title capitalization tool](https://capitalizemytitle.com/style/Chicago/).
+- Security-specific terms such as _blacklist_ or _whitelist_ are acceptable.
+
+Refer to the following guidelines:
+
+- **Methods**. To learn how to describe API methods, see [Methods](https://developers.google.com/style/api-reference-comments#methods) in [API reference code comments](https://developers.google.com/style/api-reference-comments).
```
Comment (inikolov89) [inline APIDocumentationStyleGuide.md:10]: This seems focused on "code" methods. I am wondering if they also have something for Rest APIs. I could not find something like this, so the current reference also works for me.

**Review by inikolov89 — APPROVED** [review]
Thanks!

File: APIDocumentationStyleGuide.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+- **Placeholders**. To learn more about formatting placeholders, see [Format placeholders](https://developers.google.com/style/placeholders).
+
+For more information about designing APIs, see the [API design guide](https://cloud.google.com/apis/design) by Google.
```
Comment (ana-dashuk-cobalt) [inline APIDocumentationStyleGuide.md:13]: @inikolov89 I've added this link to the API design guide, but we can remove it if you believe it's irrelevant for our product.

File: APIDocumentationStyleGuide.md (hunk 2 above)
Comment (ana-dashuk-cobalt) [reply APIDocumentationStyleGuide.md:10]: @inikolov89 Language-specific guidelines from this page also apply to RESTful APIs, for example:

- Use present tense for all descriptions.
- Start the description with a verb describing the operation.
- Capitalize the first word, and end the sentence or phrase with a period.
- Begin descriptions of non-boolean parameters with "The" or "A" if possible.

File: APIDocumentationStyleGuide.md (hunk 3 above)
Comment (inikolov89) [reply APIDocumentationStyleGuide.md:13]: I personally think it is not relevant for Rest API methods but happy to hear what the team thinks about it.

File: APIDocumentationStyleGuide.md (hunk 2 above)
Comment (inikolov89) [reply APIDocumentationStyleGuide.md:10]: Agree with that, was just asking if there is something more specific to REST. As I said, I am ok with that one. It is also helpful 👍

File: APIDocumentationStyleGuide.md (hunk 3 above)
Comment (inikolov89) [reply APIDocumentationStyleGuide.md:13]: IMO It is relevant, but this is not the place for it. I would remove it.

**Review by inikolov89 — APPROVED** [review]

**Comment by ana-dashuk-cobalt** [comment]
@inikolov89 Are we ready to merge this PR? Do we need more reviewers?

**Pull request merged**